src/datumaro/experimental/dataset.py

```diff
@@ -281,10 +304,6 @@ def convert_to_schema(
         Returns:
             A new Dataset instance with the converted schema
         """
-        # Import the converter implementations to register them
-        # ruff: noqa: F401
-        import datumaro.experimental.converters  # pyright: ignore [reportUnusedImport, reportMissingImports]
```

Comment:
This line is important. Without it, the converters won’t be registered. So we need to keep it.